player/lua/select.lua

```diff
@@ -79,15 +98,16 @@ local function format_track(track)
              and track["codec-profile"] .. " " or "") ..
             (track["demux-samplerate"] and track["demux-samplerate"] / 1000 ..
              " kHz " or "") ..
-            (track.external and "external " or "")
-        ):sub(1, -2) .. ")"
+            (track["hls-bitrate"] and track["hls-bitrate"] .. " kbps " or "")
```

Comment:
There is also `demux-bitrate` if you want to use that.

Reply:
Added.